sentry-types/src/protocol/v7.rs

```diff
@@ -1603,7 +1614,7 @@ pub struct Transaction<'a> {
     pub event_id: Uuid,
     /// The transaction name.
     #[serde(default, skip_serializing_if = "Option::is_none")]
-    pub name: Option<String>,
+    pub transaction: Option<String>,
```

Comment:
The transaction itself has a `name`, not to be confused with how the transaction gets transformed into an event before sending to Sentry. `Transaction.name` is mapped into `Event.transaction`.

Reply:
I might have been confused here since the module is called `protocol`, but these definitions don't actually map one-to-one to the payloads that are sent to the store endpoint, is that correct ? If I understand correctly there should be a piece of code (currently unimplemented) that turns a `Transaction` struct into an `Event` instead of it being serialized and submitted directly ?

Reply:
Correct, the in-memory representation is different than the over-the-wire representation.